src/app/public/modules/tabs/tabset.component.scss

```diff
@@ -45,24 +45,27 @@
     min-width: 300px;
   }
 
-  ::ng-deep {
-    .sky-dropdown-item {
-      margin: 0;
-    }
+  .sky-dropdown-item {
+    margin: 0;
+  }
 
-    .sky-btn-tab {
-      margin: 0;
-      border-radius: 0;
-      width: 100%;
-      display: flex;
+  .sky-btn-tab-wrapper {
+    width: 100%;
+    margin-right: 0 !important;
```

Comment:
I think this can be done without `!important`.

Reply:
Yep, just had to remove encapsulation on the tab-button component.